starboard/BUILD.gn

```diff
@@ -207,17 +213,79 @@ source_set("starboard_headers_only") {
   if (is_internal_build) {
     public_deps = [ "//internal/starboard/private:private_starboard_headers" ]
   }
+
+  all_dependent_configs = [
+    "//starboard/build/config:starboard",
+    "//starboard/build/config/sabi",
+    ":hacky_wnos",
+    "//base/third_party/symbolize:symbolize_config",
+  ]
+}
+
+config("hacky_wnos") {
```

Comment:
This probably should be a `TODO(b/123)`  umbrella bug for cleanups. @haozheng-cobalt can look at some of those soon, e.g. where the issues are in our code.